onnxruntime/contrib_ops/cpu/bert/multihead_attention.cc

```diff
@@ -137,6 +139,58 @@ Status MultiHeadAttention<T>::Compute(OpKernelContext* context) const {
   ORT_RETURN_IF_ERROR(MaybeTransposeToBNSHAndAddBias<T>(
       context, allocator, batch_size, num_heads_, kv_sequence_length, v_head_size, value, bias, v_bias_offset, V));
 
+  bool disableFlash;
+  {
+    char * env;
+    #ifdef _WIN32
+    _dupenv_s(&env, nullptr, "ORT_DISABLE_FLASH_ATTENTION");
+    #else
+    env = std::getenv("ORT_DISABLE_FLASH_ATTENTION");
```

Comment:
No need to getenv for each run, you can move it to constructor like
https://github.com/microsoft/onnxruntime/blob/f5b6f6dc26a55ddf7523d832ac5dc56930225264/onnxruntime/contrib_ops/cuda/bert/multihead_attention.cc#L57-L58